Shared/Samples/Select features in scene layer/README.md

```diff
@@ -0,0 +1,36 @@
+# Select feature in scene
+
+Select feature in a scene layer.
+
+![Screenshot of select feature in scene layer sample](select-features-in-scene.png)
```

Comment:
```suggestion
![Screenshot of Select feature in scene layer sample](select-features-in-scene-layer.png)
```

The README and metadata has specific formatting checks. See in `common-samples/wiki/README-Template` repo wiki for an idea.

p.s. we don't put devtopia links in public repo for security reasons, so be careful with that. No sweat if you accidentally included one - just remove it.